### PR TITLE
Dashboard demo

### PR DESCRIPTION
**Pull request by klug3**

### CODE REVIEW

File: R/app.R [hunk 1]
```diff
@@ -0,0 +1,289 @@
+library(shiny)
+library(shinydashboard)
+library(DT)
+library(tidyverse)
+library(tidyquant)
+library(quantmod)
+library(crypto2)
+library(plotly)
+library(gridExtra)
```
Comment (Suijn) [inline R/app.R:10]: We probably should get rid of `library()` in favour of `# import`

File: R/app.R [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+        )
+      )
+    )
+  )
+)
+
+# ------------------------------------------------------
+# SERVER
+#-------------------------------------------------------
+server <- function(input, output) {
```
Comment (Suijn) [inline R/app.R:213]: Maybe it would be a good idea to move `server` implementation to a different R file? What do you think?

File: R/app.R [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+
+# Extract names of top 10 cryptos
+top_10_crypto_names <- unique(top_10_crypto$name)
+  
+
+
+# ------------------------------------------------------
+# UI
+#-------------------------------------------------------
+ui <- dashboardPage(
```
Comment (Suijn) [inline R/app.R]: Maybe it would be a good idea to move ui also to a different R file? What do you think?

File: R/app.R (hunk 2 above)
Comment (klug3) [reply R/app.R:213]: good idea, but we will have to duplicate all library( ) / #import? entries or very carefully cherrypick the neccessary ones.
Decision is up to you.

File: R/app.R (hunk 1 above)
Comment (Suijn) [reply R/app.R:10]: Let's do it afterwards.

File: R/app.R (hunk 2 above)
Comment (Suijn) [reply R/app.R:213]: Let's do it in a separate pull request.

File: R/app.R (hunk 3 above)
Comment (Suijn) [reply R/app.R]: Let's do it in a separate pull request.

**Review by Suijn — APPROVED** [review]

**Pull request merged**